test/compiler/flowgraph/check_runaway.spec.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var expect = require('chai').expect;
+var juttle_test_utils = require('../../runtime/specs/juttle-test-utils');
+var check_juttle = juttle_test_utils.check_juttle;
+
+describe('Runaway program detection', function() {
+
+    function check(juttle, is_runaway) {
+        it(`${juttle} // is ` + (is_runaway ? '' : 'not') + ' a runaway', () => {
```

Comment:
this was the original way it was written and I was hoping someone else felt the same about it as I did. So I shall expand those tests out so it easier to skip/only and just read in general.